doc/nrf/device_guides/working_with_nrf/nrf70/developing/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,84 @@
+.. _ug_nrf70_developing_raw_ieee_80211_packet_reception_using_monitor_mode:
+
+Raw IEEE 802.11 packet reception using Monitor mode
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports Monitor mode, that is, listening for and receiving all IEEE 802.11 traffic on a given channel, not restricted to a certain SSID.
+With Monitor mode raw IEEE 802.11 packets can be received directly at the user application layer.
+The nRF70 device has to be configured in Monitor mode in order to operate as an 802.11 wireless packet sniffer.
+
+In Monitor mode, raw 802.11 packets received at the nRF70 Series device may be presented directly at the host application layer.
+When forwarding the received IEEE 802.11 packets to the user application, the Wi-Fi host driver will append a raw RX header to each received packet and deliver it up to the application layer using the raw socket API.
+The raw RX header holds the receive parameters for the received packet, such as data rate, receive signal strength, frequency of reception, and packet type (whether it is Legacy, HT, VHT, or HE).
+
+.. _ug_nrf70_developing_enabling_802.11_monitor_mode:
+
+Enabling Monitor mode
+*********************
+
+To use the monitor mode sniffer feature in your applications, you must enable the :kconfig:option:`CONFIG_NRF700X_RAW_DATA_RX` Kconfig option in the project configuration.
+
+.. _ug_nrf70_developing_monitor_mode_operation:
+
+Monitor mode operation
+**********************
+
+The nRF70 Series device supports IEEE 802.11 packet reception when Monitor mode is enabled in the nRF Wi-Fi driver.
+To enable the Monitor mode of operation, you can use the ``NET_REQUEST_WIFI_MODE`` network management runtime API.
+This API can be used to disable Monitor mode in the driver when IEEE 802.11 packet reception is not required by the sniffer application.
+
+If the nRF70 device is configured to operate in Monitor mode, Station mode connection is not permitted and the nRF70 Series device operates in a pure sniffer mode.
+
+When Monitor mode is enabled, you need to configure the operating channel.
+This channel will be used as the default channel on which the nRF70 device captures 802.11 packets.
+When the device operates in Monitor mode, all 802.11 packets received on the configured channel will be sent up the stack by the nRF70 Series device and the Wi-Fi driver.
+
+To set the desired channel for 802.11 packet reception, you can use the ``NET_REQUEST_WIFI_CHANNEL`` network management API.
```

Comment:
```suggestion
To set the desired channel for 802.11 packet reception, use the ``NET_REQUEST_WIFI_CHANNEL`` runtime NET-management API.
```

Reply:
It is important to clarify, also
- whether the channel configure API can be used at any time, or whether there are restrictions (e.g. when Monitor mode is not yet enabled, etc.)
- whether there needs to be a certain sequence (ENABLE, SET_CHANNEL commands) or not

Reply:
will update this table. The set channel command is not restricted to the situation wherein Monitor mode is not enabled. However, I will run this comment by the UMAC team to confirm the operation

Reply:
@ioannisg - Not sure why i wrote that comment that way. I myself found it hard to fathom what I had written. 

Set channel has to be invoked when either TX-Injection mode (station non-connected) or Monitor mode is enabled. the channel configured will be the channel on which packets are sniffed for Monitor mode and is the channel on which RAW packets are transmitted when station is not connected. Let me think how to re-word this.

Reply:
Thanks. IMHO it is valuable to clarify whether the channel can simply change in runtime, with the respective API call, independent of other configuration, or whether there needs to be a specific command sequence. This helps the users understand the steps they need to take.